client/src/app/lab-editor/editor-view/editor-view.component.ts

```diff
@@ -124,14 +126,26 @@ export class EditorViewComponent implements OnInit {
     private snackbarService: SnackbarService,
     private userService: UserService,
     private breakpointObserver: BreakpointObserver,
-    private slimLoadingBarService: SlimLoadingBarService
+    private slimLoadingBarService: SlimLoadingBarService,
+    private monacoEditorService: MonacoEditorService
   ) {}
 
   ngOnInit() {
     this.editorService.selectedTabChange.subscribe(
       tabIndex => (this.outputPanel.enabled = tabIndex === TabIndex.Console)
     );
 
+    combineLatest(
+      this.breakpointObserver.observe([Breakpoints.Tablet, Breakpoints.Web]),
+      this.editorReady$
+    ).pipe(
+      map(([state]) => state.matches ? 'off' : 'on'),
```

Comment:
Should we use `first()` to automatically unsubscribe? Not sure if we should keep the subscription open.